corehq/apps/reports/migrations/0020_tableauserver_get_reports_using_role.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.14 on 2024-08-13 19:14
+
+from django.db import migrations, models
+from corehq.apps.reports.models import TableauServer
+
+
+def _move_ff_to_setting(apps, schema_editor):
+    try:
+        from corehq.toggles import EMBED_TABLEAU_REPORT_BY_USER
+    except ImportError:
+        return
+    from corehq.toggles import EMBEDDED_TABLEAU
```

Comment:
should this also be put in the try block? So that this migration won't fail if `EMBEDDED_TABLEAU` is removed